### PR TITLE
xmind8: Update to 3.7.9

### LINKED ISSUE #94: xmind8: autoupdate issues
xmind8: couldn't match '.*>Other Platforms</h3>.*www.xmind.net/xmind/downloads/xmind-(?<mainversion>\d)\-update(?<updateversion>\d)-windows.zip' in https://www.xmind.net/download/xmind8/

https://github.com/brave-simpletons/scoop-the-business/actions/runs/3833011235/jobs/6523989191#step:3:479

The manifest may need to be updated OR this app could be deprecated since it's an old one and instead use the newest version name

### PR DESCRIPTION
**Pull request by alexchapar**
## What type of Apps related PR is this? (check all applicable)
- [ ] ✨ Add Manifest
- [ ] ♻️ Refactor
- [x] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 🚩 Other
## Description
Fix the manifest since the domain domain name changed
Update the version to the real number
## Related Tickets & Documents
Fixes #94
## [optional] What gif/emoji best describes this PR or how it makes you feel?
:alien: 🛸
### Add/Modify a description in the manifest?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Check URLs?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Update to the latest version?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Update Hashes?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Format the json manifest?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Notes in the manifest?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed
### Application Dependencies?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed
### Application Suggestions?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed
### Pre-installation steps to perform?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed
### Post-installation steps to perform?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed
### File persistence to perform?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed

### CODE REVIEW

**Comment by BotServiceAutomation** [comment]
All changes look good.

Wait for review from human collaborators.
### xmind8

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate

File: bucket/xmind8.json
Comment (Bibz87) [inline bucket/xmind8.json:7]:
```suggestion
        "url": "https://xmind.app/terms/"
```

**Review by Bibz87 — CHANGES_REQUESTED** [review]

File: bucket/xmind8.json
Comment (alexchapar) [reply bucket/xmind8.json:7]: Nice catch ! 😅

**Review by Bibz87 — APPROVED** [review]

**Pull request merged**
